app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
```

Comment:
Unless you're overriding the default behavior of Devise you do not need to generate this view. So delete this file